src/test/java/com/crud/tasks/service/SimpleEmailServiceTest.java

```diff
@@ -22,12 +24,44 @@ class SimpleEmailServiceTest {
     private JavaMailSender javaMailSender;
 
     @Test
-    void shouldSendEmail() {
+    void shouldSendEmailWithCc() {
+        //Given
+        Mail mail = Mail.builder()
+                .mailTo("test@test.com")
+                .toCc("testCC@test.com")
+                .subject("Test")
+                .message("Test Message")
+                .build();
+
+        SimpleMailMessage mailMessage = new SimpleMailMessage();
+        mailMessage.setTo(mail.getMailTo());
+        if (Optional.ofNullable(mail.getToCc()).isPresent()) {
+            mailMessage.setCc(mail.getToCc());
+        }
+        mailMessage.setSubject(mail.getSubject());
+        mailMessage.setText(mail.getMessage());
+
+        //When
+        simpleEmailService.send(mail);
+
+        //Then
+        verify(javaMailSender, times(1)).send(mailMessage);
+    }
+
+    @Test
+    void shouldSendEmailWithoutCc() {
         //Given
-        Mail mail = new Mail("test@test.com", "Test", "Test Message");
+        Mail mail = Mail.builder()
+                .mailTo("test@test.com")
+                .subject("Test")
+                .message("Test Message")
+                .build();
 
         SimpleMailMessage mailMessage = new SimpleMailMessage();
         mailMessage.setTo(mail.getMailTo());
+        if (Optional.ofNullable(mail.getToCc()).isPresent()) {
```

Comment:
jest OK, ale poszperać za możliwością wykorzystania z metody `ifPresent` - upraszcza to kod i można napisać nie pisząc w ogóle instrukcji warunkowej `if`.

Reply:
Wrzucam poprawki w commicie `674df5d`. Uwzględniają również zmiany z submodułu 23.4.